diff --git a/lib/flite-1.4/include/cst_features.h b/lib/flite-1.4/include/cst_features.h
index 373fe7e..c9b6383 100644
--- a/lib/flite-1.4/include/cst_features.h
+++ b/lib/flite-1.4/include/cst_features.h
@@ -84,6 +84,6 @@ int feat_length(const cst_features *f);
 CST_VAL_USER_TYPE_DCLS(features,cst_features)
 
 int feat_copy_into(const cst_features *from,cst_features *to);
-int feat_print(cst_file fd,const cst_features *f);
+int flite_feat_print(cst_file fd,const cst_features *f);
 
 #endif
diff --git a/lib/flite-1.4/src/synth/cst_ssml.c b/lib/flite-1.4/src/synth/cst_ssml.c
index f1e635e..b32325b 100644
--- a/lib/flite-1.4/src/synth/cst_ssml.c
+++ b/lib/flite-1.4/src/synth/cst_ssml.c
@@ -152,7 +152,7 @@ static cst_utterance *ssml_apply_tag(const char *tag,
                                      cst_features *word_feats)
 {
     printf("SSML TAG %s\n",tag);
-    feat_print(stdout,attributes);
+    flite_feat_print(stdout,attributes);
     printf("...\n");
 
     /* do stuff */
diff --git a/lib/flite-1.4/src/utils/cst_features.c b/lib/flite-1.4/src/utils/cst_features.c
index 671bb16..32b5473 100644
--- a/lib/flite-1.4/src/utils/cst_features.c
+++ b/lib/flite-1.4/src/utils/cst_features.c
@@ -258,7 +258,7 @@ int feat_copy_into(const cst_features *from,cst_features *to)
     return i;
 }
 
-int feat_print(cst_file fd,const cst_features *f)
+int flite_feat_print(cst_file fd,const cst_features *f)
 {
     cst_featvalpair *p;
     
