diff --git a/lib/googletest/googlemock/CMakeLists.txt b/lib/googletest/googlemock/CMakeLists.txt
index beb259a..b9b53f6 100644
--- a/lib/googletest/googlemock/CMakeLists.txt
+++ b/lib/googletest/googlemock/CMakeLists.txt
@@ -102,14 +102,6 @@ endif()
 
 ########################################################################
 #
-# Install rules
-install(TARGETS gmock gmock_main
-  DESTINATION lib)
-install(DIRECTORY ${gmock_SOURCE_DIR}/include/gmock
-  DESTINATION include)
-
-########################################################################
-#
 # Google Mock's own tests.
 #
 # You can skip this section if you aren't interested in testing
diff --git a/lib/googletest/googletest/CMakeLists.txt b/lib/googletest/googletest/CMakeLists.txt
index 621d0f0..f949217 100644
--- a/lib/googletest/googletest/CMakeLists.txt
+++ b/lib/googletest/googletest/CMakeLists.txt
@@ -101,14 +101,6 @@ endif()
 
 ########################################################################
 #
-# Install rules
-install(TARGETS gtest gtest_main
-  DESTINATION lib)
-install(DIRECTORY ${gtest_SOURCE_DIR}/include/gtest
-  DESTINATION include)
-
-########################################################################
-#
 # Samples on how to link user tests with gtest or gtest_main.
 #
 # They are not built by default.  To build them, set the
