diff --git a/lib/pocketsphinx-5prealpha-2015-08-05/include/pocketsphinx_export.h b/lib/pocketsphinx-5prealpha-2015-08-05/include/pocketsphinx_export.h
index 4e76f62..a82acd6 100644
--- a/lib/pocketsphinx-5prealpha-2015-08-05/include/pocketsphinx_export.h
+++ b/lib/pocketsphinx-5prealpha-2015-08-05/include/pocketsphinx_export.h
@@ -1,15 +1,7 @@
 #ifndef __POCKETSPHINX_EXPORT_H__
 #define __POCKETSPHINX_EXPORT_H__
 
-/* Win32/WinCE DLL gunk */
-#if (defined(_WIN32) || defined(_WIN32_WCE)) && !defined(_WIN32_WP) && !defined(__MINGW32__) && !defined(__CYGWIN__) && !defined(__WINSCW__) && !defined(__SYMBIAN32__)
-#ifdef POCKETSPHINX_EXPORTS /* Visual Studio */
-#define POCKETSPHINX_EXPORT __declspec(dllexport)
-#else
-#define POCKETSPHINX_EXPORT __declspec(dllimport)
-#endif
-#else /* !_WIN32 */
+// We're creating a static library. No __declspec!
 #define POCKETSPHINX_EXPORT
-#endif
 
 #endif /* __POCKETSPHINX_EXPORT_H__ */
diff --git a/lib/pocketsphinx-5prealpha-2015-08-05/model/en-us/cmudict-en-us.dict b/lib/pocketsphinx-5prealpha-2015-08-05/model/en-us/cmudict-en-us.dict
index 16eb0c4..1d130e8 100644
--- a/lib/pocketsphinx-5prealpha-2015-08-05/model/en-us/cmudict-en-us.dict
+++ b/lib/pocketsphinx-5prealpha-2015-08-05/model/en-us/cmudict-en-us.dict
@@ -134520,3 +134520,6 @@ zyuganov's Z Y UW G AA N AA V Z
 zyuganov's(2) Z UW G AA N AA V Z
 zyuganov(2) Z UW G AA N AA V
 zywicki Z IH W IH K IY
+poverty's P AA V ER T IY S
+foregrounded F AO R G R AW N D IH D
+pantries P AE N T R IY S
